2053 document featsink support redis cluster url (#2089)
risingwavelabs/risingwave-docs: docs/guides/sink-to-redis.md

| Parameter Names | Description |
| --------------- | ---------------------------------------------------------------------- |

> |redis.url | Required. The address of the Redis cluster, or a non-cluster Redis address. You can only choose one of these two types of addresses. If the address is a cluster address, it should be in the form of a JSON array, like `redis.url= '["redis://redis-server:6379/"]'`. If the address is a non-cluster address, it should be in the form of a string, like `redis.url= 'redis://redis-server:6379/'`.|

Reviewer: ```suggestion
|redis.url | Required. Choose either the Redis cluster address or a non-cluster Redis address. If the address is a cluster address, it should be in the form of a JSON array, like `redis.url= '["redis://redis-server:6379/"]'`. If the address is a non-cluster address, it should be in the form of a string, like `redis.url= 'redis://redis-server:6379/'`.|
```
